File: osis/labs/LR-7-8/block-3/main.py
# Ввод строки
s = input()
result = ""
i = 0
while i < len(s):
    if s[i] == 'x' and i + 3 <= len(s) and s[i+1:i+4] == 'abc':
        i += 1
    else:
        result += s[i]
        i += 1
print("Результат:", result)

File: osis/labs/LR-7-8/block-3/main.pas
var
  s, result: string;
  i: integer;

begin
  writeln('Введите строку:');
  readln(s);
  result := '';
  i := 1;
  while i <= length(s) do
  begin
    if (s[i] = 'x') and (i + 3 <= length(s)) and (copy(s, i + 1, 3) = 'abc') then
    begin
      i := i + 1; 
    end
    
    else
    begin
      result := result + s[i];
      i := i + 1;
    end;
  end;

  writeln('Результат: ', result);
end.
